database/seed.js

```diff
@@ -1,32 +1,37 @@
-const faker = require('faker');
-const Gallery = require('./models/gallery');
 const mongoose = require('mongoose');
-const { photoGroups } = require('./photoGroups');
+const Gallery = require('./models/gallery');
+const { arrayGenerator } = require('./dataGenerator');
 
 mongoose.connect((process.env.MONGODB_URI || 'mongodb://localhost/galleries'), { useCreateIndex: true, useNewUrlParser: true });
-const db = mongoose.connection;
```

Comment:
just fyi this didnt need to be `let` as you never reassign that, i see you changed that.  but just a comment

Reply:
<3 Thank you!